x/supplier/types/message_submit_proof_test.go

```diff
@@ -0,0 +1,40 @@
+package types
+
+import (
+	"testing"
+
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/stretchr/testify/require"
+	"pocket/testutil/sample"
+)
+
+func TestMsgSubmitProof_ValidateBasic(t *testing.T) {
```

Comment:
Need to add tests for adding more tests related to all the types. Please add `TODO(@bryanchriswhite)` for it.

See #59 for an example